schema.sql

```diff
@@ -1,5 +1,31 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+id integer,
+name text,
+date_of_birth date,
+escape_attempts integer,
+neutered boolean,
+weight_kg decimal
 );
+
+ALTER TABLE animals
+ADD COLUMN species text;
+
+
+-- Remove column species
+ALTER TABLE animals DROP COLUMN species
+-- species field column
+ALTER TABLE animals ADD species CHAR(250);
+
+-- Add column species_id which is a foreign key referencing species table
+ALTER TABLE animals
+ADD species_id INT;
+ALTER TABLE animals
+ADD CONSTRAINT fk_species_table FOREIGN KEY(species_id) REFERENCES species(id);
+
+-- Add column owner_id which is a foreign key referencing the owners table
+ALTER TABLE animals
+ADD owner_id INT;
+ALTER TABLE animals
+ADD CONSTRAINT fk_owner_table FOREIGN KEY(owner_id) REFERENCES owners(id);
```

Comment:
- [ ] Great job making the queries, however you seem to have misunderstood the tasks. You needed to create two completely new tables called `owners` and `species` here, and not add the entities to the `animals` table directly. You were supposed to create foreign keys that referenced the species and the owner id's inside the animals table. Kindly add these two tables, alter the queries so they work with the new tables and resubmit the project.
For example, the owners table can be created like below
``` SQL
CREATE TABLE owners (
   id int NOT NULL IDENTITY(1,1),
   full_name varchar(255),
   age int,
   PRIMARY KEY (id)
   );

```